src/special/package.js

```diff
@@ -0,0 +1,30 @@
+export default function parsePackageJson(content, filepath, deps) {
+  // Mark dependencies as "used" if they are used in a script in package.json.
+  if (filepath.match(new RegExp('(\\/|\\\\)package.json$'))) {  // Escaping is fun!
```

Comment:
Why not use [`path.basename`](https://api.nodejs.org/dist/latest-v6.x/docs/api/path.html#path_path_basename_path_ext)? That could avoid this interesting regexp.